src/pudl/extract/eia930.py

```diff
@@ -47,40 +47,28 @@ def process_raw(
         return df.rename(columns=self.METADATA.get_column_map(page, **partition))
 
 
-raw_eia930__all_dfs = raw_df_factory(Extractor, name="eia930")
-
-
-@asset(
-    required_resource_keys={"datastore", "dataset_settings"},
-)
-def raw_eia930__balance(raw_eia930__all_dfs):
-    """Extract raw EIA 930 balance data from CSV sheets into dataframes.
-
-    Returns:
-        An extracted EIA 930 balance dataframe.
-    """
-    return Output(value=raw_eia930__all_dfs["balance"])
-
+def raw_eia930_asset_factory(page: str):
+    """Asset factory for individual raw EIA 930 dataframes."""
 
-@asset(
-    required_resource_keys={"datastore", "dataset_settings"},
-)
-def raw_eia930__interchange(raw_eia930__all_dfs):
-    """Extract raw EIA 930 interchange data from CSV sheets into dataframes.
+    @asset(
+        name=f"raw_eia930__{page}",
+        op_tags={"memory-use": "high"},
```

Comment:
The loading of the individual EIA-930 dataframes appears to be the proximate cause of our recent failures.  These assets just read the pickled dictionary of dataframes from disk, and then immediately write the dataframes back to disk as individual pickled files. Which means each of them can spike to ~10GB of memory, and they all happen at exactly the same time, and this frequently happens to coincide with the (also) high memory FERC-714 demand matrix asset.

Reply:
I haven't dug into the 930 code but would it be appropriate to use dask here so we aren't bringing 10 GB of ram into memory?